### PR TITLE
add commitment to npc

### PR DESCRIPTION
**Pull request by Remwrath**
added commitment lvl to npc

### CODE REVIEW

File: NPC.gd [hunk 1]
```diff
@@ -1,6 +1,12 @@
 class_name NPC
 extends KinematicBody2D
 
+enum Slogan {
+	eggs,
+	milk,
+	bread
```
Comment (aaronfranke) [inline NPC.gd:7]: Try to follow the [style guide](https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html). Enum values should be UPPERCASE, similarly to the `Type` enum below.

Comment (Remwrath) [reply NPC.gd:7]: I was thinking the slogans might be displayed onscreen

Comment (aaronfranke) [reply NPC.gd:7]: Enum values aren't strings. If they are to be displayed onscreen, there would need to be a function with a match statement, something like this:

```
func slogan_string(slogan):
	match slogan:
		Slogan.EGGS:
			return "eggs"
		Slogan.MILK
			return "milk"
```

Comment (Remwrath) [reply NPC.gd]: k made it into a list

Comment (Remwrath) [reply NPC.gd]: sorry I didnt test, fixed the issues

**Pull request merged**